packages/core/src/routes/interaction/single-sign-on.ts

```diff
@@ -79,7 +79,7 @@ export default function singleSignOnRoutes<T extends IRouterParamContext>(
         connectorId: z.string(),
       }),
       body: z.record(z.unknown()),
-      status: [200, 404, 422],
+      status: [200, 404, 422, 500],
```

Comment:
Not sure but we don't add `500` to the response status manually?

Reply:
If not, it will throw an invalid response status expect [200, 404, 422] but received 500. The original 500 error will be hidden.